app/models/teacher.rb

```diff
@@ -6,4 +6,8 @@ class Teacher < ApplicationRecord
   # Validations
   validates :name,
             presence: true
+
+  validates :trn,
+            inclusion: { in: 999_999..10_000_000, message: 'TRN must be a 7 digit number' },
```

Comment:
This allows 999_999 in the range
```suggestion
            inclusion: { in: 1_000_000..10_000_000, message: 'TRN must be a 7 digit number' },
```